packages/utils/src/validate-action/validatePlayback.ts

```diff
@@ -5,7 +5,7 @@ import { Playback } from 'ontime-types';
  */
 export function validatePlayback(currentPlayback: Playback) {
   return {
-    start: currentPlayback !== Playback.Stop,
+    start: currentPlayback !== Playback.Stop && currentPlayback !== Playback.Play,
```

Comment:
@cpvalente are you OK with this?
I saw that it was possible to click the play button in the UI while something was playing and that would generate error messages in the Log, nothing was breaking but....

Reply:
yeah, I cant remember if there was a use case for still letting it through. The change sounds good